### DISCUSSION TITLE
v1.7.6

### DISCUSSION THREAD

**Discussion by aluminum-ice** [aluminum-ice/pwnagotchi#30] [Announcements]
## What's Changed
* 9b51c9c693503b723cb45bc672118b331d17d6bd Added support for the display mini hat from PR https://github.com/aluminum-ice/pwnagotchi/pull/29 (@do-ki @NeonLighting)
* 87f8b45db892ee8775ec768fd8542d4d03ae91e1 Added the mastodon and aircrackonly community plugins to the image
* a49251a5eec80de516a604c87cf4e20b5f73b29e Updated the right location to copy the kernel module for the RPiZW (e.g., /lib/modules/5.10.103+/ for RPiZW)

**Full Changelog**: https://github.com/aluminum-ice/pwnagotchi/compare/v1.7.5.1...v1.7.6

<hr /><em>This discussion was created from the release <a href='https://github.com/aluminum-ice/pwnagotchi/releases/tag/v1.7.6'>v1.7.6</a>.</em>

**Comment by wpa-2** [comment]
Does pwnagotchi auto update?

**Reply by aluminum-ice** [reply]
That is a good question. I've not tried it. I should check and see... or you let me know :-D

**Reply by wpa-2** [reply]
Sadly not the auto update plugin runs but still shows >> (v1.7.51)
I guess it might be linked to orignal repo possibly. 

[2023-09-01 20:20:17,502] [INFO] [update] checking for updates ...
[2023-09-01 20:20:20,604] [INFO] [update] done
